maestro/common/subflow-skip-chrome-welcome.yaml

```diff
@@ -1,25 +1,12 @@
 appId: ${APP_ID}
 ---
 ####### Bypass Chrome on-boarding screen #######
-# If no default browser selected, select Chrome
-- tapOn:
-    text: ".*Chrome.*"
-    optional: true
-- tapOn:
-    text: ".*ALWAYS.*"
-    optional: true
-# ACCEPT TERMS (tries id and text)
```

Comment:
this sometimes clicks on random "chrome" texts on the browser, and it's not needed anymore as Chrome is always default.